src/utils/constants.js

```diff
@@ -6,6 +6,10 @@ import logger from 'winston';
 dotenv.config();
 export const { BOT_TOKEN, MONGODB } = process.env;
 
+// Variables
+export const prefix = '++';
+export const validCmds = ['ama', 'faq', 'pins', 'salary', 'settings', 'perms'];
```

Comment:
So any new bot would have to be added here I am assuming?
Basically making the path:
person types cmd -> to parser, gets bot subCmd -> sends all substrings to subCmd for minibot to parse?

Reply:
so far, yes

Reply:
depending on how complicated this gets, the structure of the constant or of the parser may change